src/pyskindose/settings_pyskindose.py

```diff
@@ -177,9 +186,9 @@ def __init__(self, ptm_dim: dict):
         self.pad_thickness = ptm_dim['pad_thickness']
```

Comment:
Hard coded strings. Extract to constants?

Reply:
done